tests/unit/tests/PaymentGateways/Gateways/Stripe/CreditCardGatewayTest.php

```diff
@@ -10,6 +10,14 @@
  */
 class CreditCardGatewayTest extends Give_Unit_Test_Case
 {
+    public function setUp()
```

Comment:
@kjohnson yes, it is required. I edited `give_stripe_get_statement_descriptor` function which now returns statement descriptor for the stripe account instead of the global statement descriptor because this change test was failing. 

Let me know if you have any queston.